src/Engine/Audio/SoundBuffer.hpp

```diff
@@ -45,14 +45,22 @@ namespace Audio {
              * @param soundFile The sound file containing the sound.
              */
             void Load(SoundFile* soundFile);
-            
+
+            /// Get size of buffer.
+            /**
+             * @return size_t The size of the buffer.
```

Comment:
Type of the return parameter should be document, just its function. Doxygen can already determine the type of parameters.